api/Features/Topics/DeleteTopicRequest.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using MediatR;
 using Microsoft.EntityFrameworkCore;
+using Newtonsoft.Json;
```

Comment:
Is this the right one? Isnt it supposed to be System.Text.Json.Serialization?

Reply:
They both do the same thing honestly. There is a ton of overlap in functionality between Newtonsoft and System.Text.Json